src/plugins/vis_type_timeseries/server/lib/vis_data/helpers/unit_to_seconds.ts

```diff
@@ -32,49 +35,53 @@ const units = {
   y: 86400 * 7 * 4 * 12, // Leap year?
 };
 
-const sortedUnits = sortBy(Object.keys(units), (key) => units[key]);
+const sortedUnits = sortBy(Object.keys(units), (key: Unit) => units[key]);
 
-export const parseInterval = (intervalString) => {
-  let value;
-  let unit;
+export interface ParsedInterval {
```

Comment:
Why are we exporting this interface. I can't find anywhere else that is used, or maybe I am wrong 🤔

Reply:
thank you